src/data/dark.json

```
@@ -219,6 +219,79 @@
         }
       }
     },
+    "success": {
+      "text": {
+        "base": {
+          "value": "{colors.green.600}",
+          "type": "color",
+          "description": "Used as the text or fill colour for successful intention versions of components such as message, pill, or box"
+        },
+        "hover": {
+          "value": "{colors.green.700}",
+          "type": "color",
+          "description": "Used as the hovered text or fill colour for successful intention versions of components such as message, pill, or box"
+        },
+        "active": {
+          "value": "{colors.green.800}",
+          "type": "color",
+          "description": "Used as the pressed text or fill colour for successful intention versions of components such as message, pill, or box"
+        }
+      },
+      "background": {
+        "base": {
+          "value": "{colors.green.500}",
+          "type": "color",
+          "description": "Description to come."
+        },
+        "hover": {
+          "value": "{colors.green.400}",
+          "type": "color",
+          "description": "Description to come."
+        },
+        "active": {
+          "value": "{colors.green.300}",
+          "type": "color",
+          "description": "Description to come."
+        },
+        "muted": {
+          "value": "{colors.green.200}",
+          "type": "color",
+          "description": "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with successful.textBase for a subtler contrast"
```

Should these be `success`?

```suggestion
          "description": "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

---

Yes they should, although I **just** pushed the update from some edits Cath suggested, it's already reflected to be `success` :)